Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/details/SubmissionDetails.tsx

```diff
@@ -256,17 +238,55 @@ const SubmissionDetails = () => {
         [ currentSubmission, detailsHeadingText, submissionDetailsClassName ],
     );
 
-    if (isNil(currentSubmission)) {
+    const renderErrorHeading = useCallback(
+        (message: string) => (
+            <div className={styles.headingContest}>
+                <Heading
+                  type={HeadingType.primary}
+                  className={styles.contestHeading}
+                >
+                    {message}
+                </Heading>
+            </div>
+        ),
+        [],
+    );
+
+    const renderErrorMessage = useCallback(() => {
+        if (!isNil(submissionDetailsError)) {
+            return renderErrorHeading(submissionDetailsError?.detail);
+        }
+
+        if (!isNil(problemResultsError)) {
+            return renderErrorHeading(problemResultsError?.detail);
+        }
+
+        return null;
+    }, [ renderErrorHeading, submissionDetailsError, problemResultsError ]);
```

Comment:
You can place it on a new line, just like the previous comment.

Reply:
Done.